compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeWindow.uikit.kt

```diff
@@ -156,6 +161,15 @@ internal actual class ComposeWindow : UIViewController {
     private var safeArea by mutableStateOf(PlatformInsets())
     private var layoutMargins by mutableStateOf(PlatformInsets())
 
+    //invisible view to track system keyboard animation
+    private val keyboardAnimationView: UIView by lazy {
+        UIView.new()!!.apply {
+            setFrame(CGRectMake(0.0, 0.0, 0.0, 0.0))
+            hidden = true
+        }
```

Comment:
```suggestion
        UIView(CGRectMake(0.0, 0.0, 0.0, 0.0)).apply {
            hidden = true
        }
```
I don't know which variant is better.
Why are you using UIView.new()!! ?

Reply:
My vote is to avoid `!!`

Reply:
I'd avoid this `new` method. Cinterop can't derive correct nullability for it.

Reply:
I didn't notice the explicit constructor :)